src/py21cmfast/src/UsefulFunctions.c

```diff
@@ -758,11 +758,19 @@ double molecular_cooling_threshold(float z){
     return TtoM(z, 600, 1.22);
 }
 
-double lyman_werner_threshold(float z, float J_21_LW){
-    // this follows Visbal+15, which is taken as the optimal fit from Fialkov+12 which
-    // was calibrated with the simulations of Stacy+11 and Greif+11;
-    double mcrit_noLW = 3.314e7 * pow( 1.+z, -1.5);
-    return  mcrit_noLW * (1. + 22.8685 * pow(J_21_LW, 0.47));
+double lyman_werner_threshold(float z, float J_21_LW, float vcb, struct AstroParams *astro_params){
+    // correction follows Schauer+20, fit jointly to LW feedback and relative velocities. They find weaker effect of LW feedback than before (Stacy+11, Greif+11, etc.) due to HII self shielding.
+    double mcrit_noLW = 3.314e7 * pow( 1.+z, -1.5);// this follows Visbal+15, which is taken as the optimal fit from Fialkov+12 which was calibrated with the simulations of Stacy+11 and Greif+11;
+
+    double f_LW = 1.0 + astro_params->A_LW * pow(J_21_LW, astro_params->BETA_LW);
```

Comment:
The reason those tests with MINI_HALOs on failed is that feedback strength is now different from previous. By default when vcb=0, f_LW is still !=(1. + 22.8685 * pow(J_21_LW, 0.47)). 

I dont have a strong opition here, but either we support the previous strong feedback prescription (e.g. have a flag or sth) or we update the test files. Supporting previous version allow users to easily reprocude previous results, but it's also kind of redundant to do so considering it's just different ways of treating LW feedback...

I'll let other @21cmfast/admins decide

Reply:
Good point. As you say, the old fit can be obtained by setting astro_params->A_LW=22.8685 and astro_params->BETA_LW=0.47, that is the old Macachek 2001 result. Maybe we can set it to that just for the test?

Reply:
Ah, no idea why I didn't realize that... Yeah, in this case, resetting the default will do

Reply:
It may make sense to update the "golden" results to the new MCG feedback prescription, which is much lower bc of proper H_2 self shielding. I have no strong feelings about this, since it's a test. @21cmfast/admins?